packages/fast-components-react-base/src/checkbox/checkbox.tsx

```diff
@@ -87,6 +90,36 @@ class Checkbox extends Foundation<ICheckboxHandledProps & ICheckboxManagedClasse
 
         return super.generateClassNames(classes);
     }
+
+    /**
+     * Stores HTML tag for use in render
+     */
+    private get tag(): string {
+        return CheckboxHTMLTags[this.props.tag] || CheckboxHTMLTags.label;
+    }
+
+    /**
+     * Apply indeterminate state to items that are indeterminate.
+     * This method should be called after render because it relies on element references.
+     */
+    private applyIndeterminateState(): void {
+         if (this.props.indeterminate !== undefined && this.inputRef.current && this.state.checked) {
+            this.inputRef.current.indeterminate = this.props.indeterminate;
+        }
+    }
+
+    /**
+     * Handles onChange as a controlled component
+     */
+    private handleCheckboxChange = (e: any): void => {
```

Comment:
can you type the `e` param accurately? I think it should be `React.ChangeEvent<HTMLInputElement>`

Reply:
Sure